### DISCUSSION TITLE
Adding support for Cisco 8000 simulator

### DISCUSSION THREAD

**Discussion by rskorka** [srl-labs/containerlab#1212] [Q&A] [answered]
I would like to add support for Cisco 8000 simulator to containerlab. How do I contribute the changes?

**Comment by hellt** [comment, marked as answer]
Hi @rskorka 
I have replied in the meantime in discord. In general, I understand that you have a working implementation already, so just file a PR and we will review it

**Reply by rskorka** [reply]
PR has been raised. Thank you.
